status.go

```diff
@@ -183,22 +179,10 @@ func statusLine(statusCode int) []byte {
 	return statusLines[statusCode]
 }
 
-// invalidStatusLine return status line of which bigger than statusMessageMax(511)
+// invalidStatusLine return status line of which
+// smaller than 0 or
+// bigger than statusMessageMax(511)
```

Comment:
```suggestion
```

One last pedantic thing. I don't think the comment makes much sense so better just remove it.